src/components/shared/slide-error.tsx

```diff
@@ -1,17 +1,17 @@
 import React from "react";
 
 const SlideError = ({ children, title, image }) => {
-    return (
-        <section className="slide">
-        <div className="container bordered indented-container">
+  return (
+    <section className="slide">
+      <div className="container bordered indented-container">
         <div className="slide-body slide-error-body">
-        <h1 className="slide-title">{title}</h1>
-        <hr className="underline" />
-        {children}
+          <h1 className="slide-title">{title}</h1>
+          <hr className="underline" />
```

Comment:
can we styled-componentize this stuff too?

Reply:
Will do.